core/src/main/scala/org/apache/spark/scheduler/cluster/CoarseGrainedClusterMessage.scala

```diff
@@ -49,6 +49,10 @@ private[spark] object CoarseGrainedClusterMessages {
   case class KillExecutorsOnHost(host: String)
     extends CoarseGrainedClusterMessage
 
+  case class RefreshExecutors(msg: Map[String, String]) extends CoarseGrainedClusterMessage
+
+  case class RefreshExecutor(msg: Map[String, String]) extends CoarseGrainedClusterMessage
```

Comment:
I gave a generating name to this message as this can hold a map of message values and in the future,  this can be reused,  if any other message needs to be passed to the executor. 

 If that is not required, I can change this to a specific message name. 
eg: `UpdateExecutorsLogLevel(msg: String)`  & `UpdateExecutorLogLevel(msg: String)`
    OR 
`RefreshExecutorsLogLevel(msg: String)` & `RefreshExecutorLogLevel(msg: String)`

Please give your suggestions

Reply:
Update sounds better to me.

Reply:
+1 that the messasge name is too generic. It should reflect the intent of the caller. I would prefer `Update*` as well

Reply:
Changed to `UpdateExecutorsLogLevel(msg: String)` & `UpdateExecutorLogLevel(msg: String)`